src/main/java/org/broadinstitute/hellbender/utils/read/SAMRecordToGATKReadAdapter.java

```diff
@@ -21,12 +23,15 @@
  */
 public final class SAMRecordToGATKReadAdapter implements GATKRead, Serializable {
     private static final long serialVersionUID = 1L;
+    private final static long uuidHighWord = new Random().nextLong();
+    private final static AtomicLong uuidLowWord = new AtomicLong(0);
 
     private final SAMRecord samRecord;
     private final UUID uuid;
 
     public SAMRecordToGATKReadAdapter( final SAMRecord samRecord ) {
-        this(samRecord, UUID.randomUUID());
+        // this is 100x faster than UUID.randomUUID()
+        this(samRecord, new UUID(uuidHighWord, uuidLowWord.incrementAndGet()));
```

Comment:
Can you please make the equivalent change in the `GoogleGenomicsReadToGATKReadAdapter` for the sake of consistency?